rendezvous/README.md

```diff
@@ -206,6 +254,8 @@ message Message {
     E_INVALID_SIGNED_PEER_RECORD  = 101;
     E_INVALID_TTL                 = 102;
     E_INVALID_COOKIE              = 103;
+    E_POW_REQUIRED                = 104;
+    E_DIFFICULTY_TOO_LOW          = 105;
```

Comment:
Thinking of dropping this one because it is essentially the same message as `E_POW_REQUIRED`.